### ISSUE TITLE
Replicating the results

### ISSUE THREAD

**Issue by Ellenisawake** [mil-tokyo/MCD_DA#15]
Does anybody succeded in getting the results as the paper claimed?
So far by running the VisDA experiment with ResNet101 using default settings I get this:
![screenshot from 2019-01-28 10-39-57](https://user-images.githubusercontent.com/12854625/51831261-9429c300-22e9-11e9-87f4-97078ea522f9.png)
Mean class accuracy 62.54% (v.s 71.9 reported in paper)

**Comment by Ellenisawake** [comment]
Updated results on VisDA17 classification (**70.11%**):
![screenshot from 2019-01-31 10-03-49](https://user-images.githubusercontent.com/12854625/52046848-a56c0d00-253f-11e9-9ed9-cd06509f187e.png)
Difference is that previous result was obtained running the code under PyTorch 0.4, this one is using **PyTorch 0.3**. And running evaluation in either 0.3 or 0.4 gives the same accuracy. Suspect that behavior of some function(s) changed dramastically from 0.3 to 0.4. 
Thanks to the author for pointing out that:)

**Comment by guantinglin** [comment]
Hi, @Ellenisawake 
do you have any idea what makes this performance difference betewwn pytorch 0.3 and 0.4?

**Comment by jiaojiao-Chen** [comment]
Hi, @Ellenisawake
do you use the code the author applied with VisDa classification directly? I couldn't understand the variable of 'val' in the test function, can you tell me your understand?

**Comment by Ellenisawake** [comment]
> Hi, @Ellenisawake
> do you have any idea what makes this performance difference betewwn pytorch 0.3 and 0.4?

I didn't check in detail. Suggest that you just use 0.3 for running it or adding anything on top. As PyTorch has already evolved to v1.0, personally don't think there's much significance in figuring out version conflicts between 0.3 & 0.4.

**Comment by Ellenisawake** [comment]
> Hi, @Ellenisawake
> do you use the code the author applied with VisDa classification directly? I couldn't understand the variable of 'val' in the test function, can you tell me your understand?

I think 'val' is just a flag used to distinguish a test run and a validation run within training. 
'''
if not val and value > 60:
    torch.save(F1.state_dict(), save_path+'_'+args.resnet+'_'+str(value)+'_'+'F1.pth')
'''
Here the author only wants the model saved when in validation and accuracy >60.
